assets/js/guides.js

```diff
@@ -120,4 +120,19 @@ $(document).ready(function () {
     ga('send', 'event', location.pathname, 'subscribe-cta');
   })
 
+  //Remove link from parent navigation item and add classes when clicked
+  $.each($("#toc ul > li"), function (index, obj) {
+    if ($(obj).has("ul").length) {
+      const parentNavItem = $(obj).has("ul");
+
+      parentNavItem.find("a:first").removeAttr("href");
```

Comment:
Why remove? Seems like you're already overriding the click behavior below anyway.

Reply:
Because on click, if the link still exists, it scrolls to that position. So I'm removing the link while clicking removes existing classes